### DISCUSSION TITLE
LaserGrbl settings guide translated into Dutch

### DISCUSSION THREAD

**Discussion by Tribo-2dot0** [arkypita/LaserGRBL#2002] [General]
In the FB group some Dutch people asked for the guide in their language. I did a translation via google translator with the revision of Paul Robbecke.
[Grbl-Instellingen-DUT.pdf](https://github.com/arkypita/LaserGRBL/files/10458113/Grbl-Instellingen-DUT.pdf)

**Comment by arkypita** [comment]
good, thank you!

**Reply by arkypita** [reply]
shared here: https://lasergrbl.com/configuration/
